install.sh

```diff
@@ -350,6 +360,7 @@ FORMAT=tar.gz
 OS=$(uname_os)
 ARCH=$(uname_arch)
 PREFIX="$OWNER/$REPO"
+RELEASE_V1_UPPERBOUND="v1.1.10"
```

Comment:
Will we have to update this here and in `install.ps1` if we make a new v1 release? Or is the plan to never make a new v1 release

Reply:
If we have to change it, maybe it would be better to name it `RELEASE_V1_LATEST` so we're more likely to notice it